fluss-flink/fluss-flink-common/src/main/java/com/alibaba/fluss/flink/utils/FlussRowToFlinkRowConverter.java

```diff
@@ -42,7 +42,8 @@
  * <p>Note: fluss-datalake-tiering also contains the same class, we need to keep them in sync if we
  * modify this class.
  */
-public class FlussRowToFlinkRowConverter {
+public class FlussRowToFlinkRowConverter implements Serializable {
```

Comment:
it doesn't need to implement `Serializable` now, as it is initialized in `open()`